src/Service/S3/src/Input/PutObjectRequest.php

```diff
@@ -53,9 +53,9 @@ final class PutObjectRequest extends Input
 
     /**
      * Specifies presentational information for the object. For more information, see
-     * https://www.rfc-editor.org/rfc/rfc6266#section-4.
+     * http://www.w3.org/Protocols/rfc2616/rfc2616-sec19.html#sec19.5.1.
      *
-     * @see https://www.rfc-editor.org/rfc/rfc6266#section-4
+     * @see http://www.w3.org/Protocols/rfc2616/rfc2616-sec19.html#sec19.5.1
```

Comment:
😿

Reply:
http instead of https

Reply:
:shrug: This is extracted from AWS metadata.